Makefile

```diff
@@ -0,0 +1,22 @@
+# Disable all the default make stuff
+MAKEFLAGS += --no-builtin-rules
+.SUFFIXES:
+
+.PHONY: default
+default:
+	# Please provide a valid make target
+
+## Update score schema
+.PHONY: update-schema
+update-schema:
+	rm -fv schema/files/score-v1b1.json.modified
+	C=$(shell git rev-parse HEAD); git subtree pull --prefix schema/files git@github.com:score-spec/schema.git main --squash -m "chore: updated score specification"; \
+		if git rev-parse HEAD | grep -v $$C; then git commit --amend -s --no-edit; fi
+
+## Generate types
+.PHONY: generate
+generate:
+# Unfortunately struct generators don't know how to handle mixed properties and additional properties so we have to strip these out before we generate the structs.
```

Comment:
Looks like there is already a PR https://github.com/omissis/go-jsonschema/pull/189, but this should be fine to unblock this 👍

Reply:
I'll keep an eye on that PR, it would definitely make things a little easier.